tests/test_files_directory/nested_directory/build_test_merged_file.json

```diff
@@ -1,7 +1,3 @@
 {
-    "Hello": "There",
-    "UhOh": "This",
-    "OneIs": "Nested",
-    "UhOh2": "This",
-    "OneIs2": "NestedAlso"
+    "Hello": "There"
```

Comment:
This file is used in a unit test which merges the original contents with the key you see removed here. it then saves the file and checks it as part of the test. Meaning after every test run, I have to remember to clean out the file and set it back to it's original form. A bad test we should certainly improve when we look at improving testing